cpp/ql/src/semmle/code/cpp/ir/implementation/unaliased_ssa/internal/SSAConstruction.qll

```diff
@@ -178,6 +178,22 @@ private module Cached {
     )
   }
 
+  /**
+   * Holds if the `ChiPartialOperand` totally, but not exactly, overlaps with the `ChiTotalOperand`.
```

Comment:
This should be "partially overlaps with", I believe. A "total" overlap overwrites all bits but could be a larger write or with a different type, an "exact" overlap writes with the same type and overwrites all bits.

Reply:
Good point. I've fixed this in c86fc223b938129421f2e9a5535b7d3ee87ffd40.